app/controllers/videogames_controller.rb

```diff
@@ -0,0 +1,71 @@
+class VideogamesController < ApplicationController
+  before_action :authenticate_user!, except: %i[index]
+
+  def index
+    @videogames = Videogame.all.order(id: :desc)
+    render json: @videogames
```

Comment:
Consider Error Handling here as well (as seen in `create` and `destroy`)

Reply:
Hey there, the index endpoint is a public view so it's not required neither authentication nor authorization